### PR TITLE
Declarative Breadcrumb Navigation in Ember.js

### PR DESCRIPTION
**Pull request by poteto**
Will merge tomorrow (June 2nd)

### CODE REVIEW

File: source/posts/2015-06-02-ember-crumbly.md [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+layout: post
+title: 'Declarative Breadcrumb Navigation in Ember.js'
+twitter: 'sugarpirate_'
+github: 'poteto'
+author: 'Lauren Tan'
+tags: ember, addon, javascript
+social: true
+comments: true
+published: true
+summary: "Breadcrumb navigation isn't a new concept, in fact, it's usefulness has been endlessly debated about by UX designers all over. Today, I won't get into the nitty gritty on whether or not your app should include one, but I'd like to share an addon I built for a project I'm working on that required it. This is a simple Component that is placed once in your application, and then generates a dynamic breadcrumb by looking up the current route hierarchy. The addon has a simple declarative API, which makes integration with your app super easy."
```
Comment (bcardarella) [inline source/posts/2015-06-02-ember-crumbly.md]: The summaries should be shorter. Longer summaries tend the look odd with how we're using them (in the index page).

Comment (poteto) [inline source/posts/2015-06-02-ember-crumbly.md]: I shortened it, let me know if it's still too long

File: source/posts/2015-06-02-ember-crumbly.md [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+{{bread-crumbs linkable=true outputStyle="foundation"}}
+```
+
+And the route:
+
+```js
+// foo/route.js
+export default Ember.Route.extend({
+  breadCrumb: {
+    title: 'Animals'
```
Comment (bcardarella) [inline source/posts/2015-06-02-ember-crumbly.md]: not a critique but I'm wondering if the breadcrumb titles could be taken from a library like https://github.com/kimroen/ember-cli-document-title

Comment (poteto) [inline source/posts/2015-06-02-ember-crumbly.md]: Hmm, yeah definitely could! I will investigate

**Comment by bcardarella** [comment]
:+1:

**Comment by danmcclain** [comment]
I'll buffer a tweet on the dockyard account for 1pm tomorrow

**Pull request merged**